docs/dev/tools/cli.md

```diff
@@ -1190,6 +1191,24 @@ viam.service.vision.v1.VisionService.GetClassificationsFromCamera
 
 # restart a part of a specified machine
 viam machines part restart --machine=123 --part=456
+
+# Copy a single file to the machine with a new name:
```

Comment:
Sounds ambiguous as to whether it's the same machine name or file name or directory name. Same comment on the two below.